bundles/org.openhab.binding.modbus.sunspec/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.modbus.sunspec-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>file:${basedirRoot}/bundles/org.openhab.binding.modbus/target/feature/feature.xml</repository>
+
+    <feature name="openhab-binding-modbus-sunspec" description="SunSpec Bundle" version="${project.version}">
```

Comment:
Should this be installed together with the modbus bundle (like the different bluetooth and mqtt bundles are installed together) or should it be stand-alone?

Reply:
This must be installed with the modbus bundle. It does not work alone at all, because I tried to avoid reimplementing the modbus functionalities.
AFAIK there is no way yet to force the dependency on OH but it would be very welcome here.

Reply:
I was thinking of the other way: should everyone installing the modbus bundle automatically receive this one? If that‘s the case we would have to change the feature generation.

Reply:
Modbus is a general protocol popular among industrial grade devices and professional home automation.
The basic idea is that we could have a bundle (modbus) that is flexible but needs a decent knowledge from the user, and create higher level bundles for special devices that are more user friendly.
But at the moment this bundle is the only as far as I know.

Pros: bundling this with modbus will make the life of users more simple at the moment
Cons: in the future this _might_ make the modbus bundle pull in lots of unnecessary bundles.

I'm pretty much an outsider in this community, so I won't be able to decide this, so I will leave this to you.

Also I will need help how to change the feature.xml if we decide to install this with the modbus bundle.

Reply:
We can decide that later and adjust if we decide to bundle them for OH3.